tests/snapshots/src/main/generated/com/airbnb/epoxy/AsyncEpoxyController.java

```diff
@@ -32,7 +32,6 @@
   public AsyncEpoxyController() {
 //       ^^^^^^^^^^^^^^^^^^^^ definition com/airbnb/epoxy/AsyncEpoxyController#`<init>`(). public AsyncEpoxyController()
     this(true);
-//  ^^^^ reference com/airbnb/epoxy/AsyncEpoxyController#`<init>`(+1).
```

Comment:
Hmm, I think we should continue to emit references to constructors. It's only when the reference points to a type symbol when we want to skip it. IntelliJ support goto def on `this()` when it points to a constructor.

Reply:
Good catch! Easy to miss this amongst the noise 🙂

Reply:
Fixed :+1: